### DISCUSSION TITLE
ScriptReference/EditorWindow.Repaint

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#655] [Notes]
# ScriptReference/EditorWindow.Repaint

https://docs.unity3d.com/ScriptReference/EditorWindow.Repaint.html

**Comment by andreasIsCoding** [comment]
If you need to repaint the whole editor, there is an undocumented api:
`InternalEditorUtility.RepaintAllViews ();`
It's public though, so you don't need reflection to access it.
